### ISSUE TITLE
Detect when Solr is not accessible

### ISSUE THREAD

**Issue by nichtich** [pkiraly/qa-catalogue-web#115]
There should be a visible warning when Solr is not accessible. By now warnings are only logged but the user interface shows same result as empty Solr collection.

**Comment by pkiraly** [comment]
I agree, and was thinking about the same. Maybe a sample cron script would be also attached to the installation package and the README.

**Comment by nichtich** [comment]
There is method `isCoreAvailable` in `classes/BaseTab.php`, this seems to do a check but the result is not always shown in the UI.

**Comment by pkiraly** [comment]
estimation: 4 hours

**Comment by pkiraly** [comment]
@nichtich Is this feature completed?

**Comment by nichtich** [comment]
An error message is shown when Solr is not available, so this feature is complete for data search tab. The other tabs (completeness, issues, classifications, authorities) tab work without Solr index so no need to show error for these. I've just modified terms tab to also shown an error message but collocation tab does not indicate when Solr is down.

I've neither tested with MARC data where `ControlFields.php` is used - this will also break silently if Solr is not accessible.
